docs/reference/commandline/dockerd.md

```diff
@@ -740,6 +740,19 @@ to add multiple lower directory support for OverlayFS. This option should
 only be used after verifying this support exists in the kernel. Applying
 this option on a kernel without this support will cause failures on mount.
 
+##### `overlay2.size`
```

Comment:
`dockerd` itself will stay here (at least for the time being), only the client CLI is moved to the docker/cli repo